src/backend/joanie/core/models/products.py

```diff
@@ -378,6 +378,16 @@ def find_installments(self, due_date):
             .filter(payment_schedule__contains=[{"due_date": due_date.isoformat()}])
         )
 
+    def find_today_installments(self):
+        """Retrieve orders with a payment due today."""
+        return (
+            super()
+            .get_queryset()
+            .filter(
+                payment_schedule__contains=[{"due_date": timezone.now().isoformat()}]
```

Comment:
No need to remove datetime to keep only date ?

Reply:
I think I'll move the related commit to the next PR.